doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md

```diff
@@ -0,0 +1,2578 @@
+09.06.2015 |  0.1    |   Initial version | Meril Vaht
+21.08.2015 |  0.2    |   Updated version | Meril Vaht
+11.09.2015 |  0.3    |   Updated version | Meril Vaht
+16.09.2015 |  0.4    |   Updated version | Meril Vaht
+17.09.2015 |  0.5    |   Comments and editorial changes made | Riin Saarmäe
+17.09.2015 |  0.6    |   Comments and editorial changes accepted | Meril Vaht
+20.09.2015 |  1.0    |   Editorial changes made | Imbi Nõgisto
+27.09.2015 |  1.1    |   Use cases for managing an X-Road member's global group membership records added. *Scope* element renamed to *System*. | Riin Saarmäe
+23.10.2015 |  1.2    |   User renamed to SS/CS administrator; updated: UC SERVICE\_11, XTE-78, XTE-161. | Meril Vaht
+26.10.2015 |  1.3    |   UC SERVICE\_09 updated according to system developments. | Meril Vaht
+27.10.2015 |  1.4    |   Editorial changes made | Meril Vaht
+23.12.2015 |  1.5    |   XTE-117 – WSDL validator (UC SERVICE\_08, UC SERVICE\_09, UC SERVICE\_14 updated; UC SERVICE\_44 added) | Meril Vaht
+29.12.2015 |  1.6    |   Editorial changes made | Meril Vaht
+29.08.2017 |  1.7    |  Changed documentation type from docx to md file |   Lasse Matikainen
+
+<!-- toc -->
+
+Table of Contents
+
+- [1 Introduction](#1-introduction)
+  * [1.1 Purpose](#11-purpose)
+  * [1.2 Terms and Abbreviations](#12-terms-and-abbreviations)
+  * [1.3 References](#13-references)
+- [2 Overview](#2-overview)
+- [3 Use Case Model](#3-use-case-model)
+  * [3.1 Security server](#31-security-server)
+    * [3.1.1 Actors](#311-actors)
+    * [3.1.2 UC SERVICE\_01: View the Service Clients of a Security Server Client](#312-uc-service_01-view-the-service-clients-of-a-security-server-client)
+    * [3.1.3 UC SERVICE\_02: View the Access Rights of a Service Client](#313-uc-service_02-view-the-access-rights-of-a-service-client)
+    * [3.1.4 UC SERVICE\_03: Add a Service Client to a Security Server Client](#314-uc-service_03-add-a-service-client-to-a-security-server-client)
+    * [3.1.5 UC SERVICE\_04: Add Access Rights for a Service Client](#315-uc-service_04-add-access-rights-for-a-service-client)
+    * [3.1.6 UC SERVICE\_05: Remove Access Rights from a Service Client](#316-uc-service_05-remove-access-rights-from-a-service-client)
+    * [3.1.7 UC SERVICE\_06: View the WSDLs of a Security Server Client](#317-uc-service_06-view-the-wsdls-of-a-security-server-client)
+    * [3.1.8 UC SERVICE\_07: View the Services of a Security Server Client](#318-uc-service_07-view-the-services-of-a-security-server-client)
+    * [3.1.9 UC SERVICE\_08: Add a WSDL to a Security Server Client](#319-uc-service_08-add-a-wsdl-to-a-security-server-client)
+    * [3.1.10 UC SERVICE\_09: Edit the Address of a WSDL](#3110-uc-service_09-edit-the-address-of-a-wsdl)
+    * [3.1.11 UC SERVICE\_10: Download and Parse WSDL](#3111-uc-service_10-download-and-parse-wsdl)
+    * [3.1.12 UC SERVICE\_44: Validate a WSDL](#3112-uc-service_44-validate-a-wsdl)
```

Comment:
Numbered links broken from 3.1.12 to 3.1.31

Reply:
This is still an issue.

Reply:
https://github.com/vrk-kpa/X-Road/blob/PVAYLADEV-878/doc/UseCases/uc-service_x-road_use_case_model_for_service_management_1.6_Y-883-3.md
It seems to be working

Reply:
Hmm, somehow saw a version where it didn't, but yes, now it works.